packages/sdk/src/lib/dataProtectorSharing/consumeProtectedData.ts

```diff
@@ -214,6 +215,7 @@ export const consumeProtectedData = async ({
     const { result } = await getResultFromCompletedTask({
       iexec,
       taskId,
+      path,
```

Comment:
Simply give this 'path' param to `getResultFromCompletedTask()` that knows what to do with it.